wdl/tasks/sed.wdl

```diff
@@ -28,3 +28,32 @@ task remove_trailing_whitespaces {
         disks: resources.disks
     }
 }
+
+
+task remove_first_n_lines {
+    input {
+        File input_file
```

Comment:
input_file -> in?

Reply:
yeah, I suppose it is quite clear from the type.

Reply:
changed the other task in `sed.wdl` as well.